### ISSUE TITLE
Website should score 100% for accessibilty on Lighthouse

### ISSUE THREAD

**Issue by craig-dsilva** [craig-dsilva/project-days-calendar#6]
When a lighthouse report is generated, the accessibility score on the Lighthouse should be 100%
